### PR TITLE
Improve Choice Validation Constraint Example

### PR DESCRIPTION
**Pull request by huebs**
Example came off sounding transphobic. Added "other" option.

### CODE REVIEW

File: book/validation.rst [hunk 1]
```diff
@@ -329,7 +329,7 @@ Some constraints, like :doc:`NotBlank </reference/constraints/NotBlank>`,
 are simple whereas others, like the :doc:`Choice </reference/constraints/Choice>`
 constraint, have several configuration options available. Suppose that the
 ``Author`` class has another property called ``gender`` that can be set to either
-"male" or "female":
+"male", "female", or "other":
```
Comment (xabbuh) [inline book/validation.rst]: Can you please remove the comma in front of "or"? We don't use serial commas in the documentation.

**Comment by dupuchba** [comment]
This is a funny one :-)

**Comment by xabbuh** [comment]
:+1:

**Comment by weaverryan** [comment]
Haha, indeed! With an error message of "Choose a valid gender." - that was certainly not the intention :). Thanks Ben!

**Pull request closed without merging**